CC/src/DgmOctree.cpp

```diff
@@ -1,19 +1,19 @@
 //##########################################################################
-//#                                                                        #
-//#                               CCLIB                                    #
-//#                                                                        #
-//#  This program is free software; you can redistribute it and/or modify  #
-//#  it under the terms of the GNU Library General Public License as       #
-//#  published by the Free Software Foundation; version 2 or later of the  #
-//#  License.                                                              #
-//#                                                                        #
-//#  This program is distributed in the hope that it will be useful,       #
-//#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
-//#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the          #
-//#  GNU General Public License for more details.                          #
-//#                                                                        #
-//#          COPYRIGHT: EDF R&D / TELECOM ParisTech (ENST-TSI)             #
-//#                                                                        #
+//#																		   #
```

Comment:
Maybe not the headers?

Reply:
They actually line up properly in editors - just not on GitHub.

(Also - it would be cleaner to get rid of the trailing '#' anyways :-) )

Reply:
I wish there were a GitHub setting for tab display size...